src/Tickets/Emails/Email/Failed_Order.php

```diff
@@ -0,0 +1,402 @@
+		return false;
+	}
+
+	/**
+	 * Get email title.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email title.
+	 */
+	public function get_title(): string {
+		return esc_html__( 'Failed Order', 'event-tickets' );
+	}
+
+	/**
+	 * Get email recipient.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email heading.
+	 */
+	public function get_recipient(): string {
+		$recipient = tribe_get_option( static::$option_recipient, $this->get_default_recipient() );
+
+		/**
+		 * Allow filtering the email recipient for failed orders.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email recipient.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$recipient = apply_filters( 'tec_tickets_emails_failed_order_recipient', $recipient, self::$id, $this->template );
+
+		/**
+		 * Allow filtering the email recipient globally.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email recipient.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$recipient = apply_filters( 'tec_tickets_emails_recipient', $recipient, self::$id, $this->template );
+
+		return $this->format_string( $recipient );
+	}
+
+	/**
+	 * Get default email recipient.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_default_recipient() {
+		return get_option( 'admin_email' );
+	}
+
+	/**
+	 * Get email heading.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email heading.
+	 */
+	public function get_heading(): string {
+		$heading = tribe_get_option( static::$option_heading, $this->get_default_heading() );
+
+		/**
+		 * Allow filtering the email heading for Failed Order.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email heading.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$heading = apply_filters( 'tec_tickets_emails_failed_order_heading', $heading, self::$id, $this->template );
+
+		/**
+		 * Allow filtering the email heading globally.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email heading.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$heading = apply_filters( 'tec_tickets_emails_heading', $heading, self::$id, $this->template );
+
+		return $this->format_string( $heading );
+	}
+
+	/**
+	 * Get default email heading.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_default_heading() {
+		return esc_html__( 'Failed order: #{order_number}', 'event-tickets' );
+	}
+
+	/**
+	 * Get email subject.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email subject.
+	 */
+	public function get_subject(): string {
+		$subject = tribe_get_option( static::$option_subject, $this->get_default_subject() );
+
+		$subject = $this->format_string( $subject );
```

Comment:
We should probably format the string later, as they will probably want to use the placeholders if they apply filters.